[Outlook] (event-based activation) Reorganize documentation (#4324)
OfficeDev/office-js-docs-pr: docs/outlook/autolaunch.md

---
title: Configure your Outlook add-in for event-based activation
description: Learn how to configure your Outlook add-in for event-based activation.
ms.date: 12/19/2023
ms.topic: concept-article
ms.localizationpriority: medium
---

# Configure your Outlook add-in for event-based activation

> Without the event-based activation feature, a user has to explicitly launch an add-in to complete their tasks. This feature enables your add-in to run tasks when certain events occur, such as automatically updating a mail item's attachments when the recipients have changed or checking for a particular attachment before a mail item is sent. You can also integrate event-based activation with the task pane and function commands.

Reviewer: Suggestion to frame this in a way that showcases what the user can do.
```suggestion
The event-based activation feature lets a user use your add-in to complete their tasks without explicitly launching the add-in. Your add-in will run tasks when certain events occur, such as automatically update a mail item's attachments when the recipients change or check for a particular attachment before a mail item is sent. You can also integrate event-based activation with the task pane and function commands.
```
